spec/features/state_file/complete_intake_spec.rb

```diff
@@ -422,6 +422,18 @@
       click_on I18n.t("state_file.questions.esign_declaration.edit.submit")
 
       expect(page).to have_text I18n.t("state_file.questions.submission_confirmation.edit.title", state_name: "North Carolina", filing_year: filing_year)
+
+      click_on "Main XML Doc"
+
+      expect(page.body).to include('efile:ReturnState') # QUESTION: not sure what the significance of this tag is for NC
```

Comment:
@arinchoi03 were you able to find out what this tag is for? is this comment still needed?

Reply:
@mpidcock will follow up if we need this for all states/some states/no states :)